test/master-playlist-controller.test.js

```diff
@@ -70,6 +71,9 @@ QUnit.module('MasterPlaylistController', {
     this.requests = this.env.requests;
     this.mse = useFakeMediaSource();
 
+    this.originalSelectPlaylist = Hls.STANDARD_PLAYLIST_SELECTOR;
+    Hls.STANDARD_PLAYLIST_SELECTOR = movingAverageBandwidthSelector();
```

Comment:
Yes, but because of the way the closure works, if we don't do this between tests, each test will be using the moving average of all the previous tests